tiledb/sm/query/readers/dense_reader.h

```diff
@@ -190,6 +190,7 @@ class DenseReader : public ReaderBase, public IQueryStrategy {
   template <class DimType>
   tuple<uint64_t, std::vector<ResultTile*>> compute_result_tiles(
       const std::vector<std::string>& names,
+      std::optional<ThreadPool::Task>& compute_task,
```

Comment:
Maybe a bikeshedding issue -- but is this the right place in the parameter list for the `compute_task` argument (splitting names and condition_names)?